pkg/tracer/tcptracer-ebpf.go

```diff
@@ -0,0 +1,235 @@
+// Code generated by go-bindata.
```

Comment:
Alright, then I guess it's fine, although you could always use go-generate and do it on the fly (even for vendored packages).